RNTester/Podfile

```diff
@@ -17,6 +52,12 @@ def pods()
   # Additional Pods which aren't included in the default Podfile
   pod 'React-ART', :path => '../Libraries/ART'
   pod 'React-RCTPushNotification', :path => '../Libraries/PushNotificationIOS'
+  
+  # For enabling Flipper.
+  # Note that if you use_frameworks!, flipper will no work.
```

Comment:
Why does it not work as framework?

Reply:
I do not have enough context to know why. I went off the Flipper docs.

Maybe the Flipper team could give us context.
cc @passy @rickhanlonii @axe-fb

Reply:
I’m still naive about Flipper. Could you point me to those docs where it states that?

Reply:
Yup! Check out the Podfile example here: https://fbflipper.com/docs/getting-started.html#setup-your-ios-app

Reply:
Also cc @fkgozali for use_framework implications

Reply:
I'm a total n00b when it comes to Cocoapods, unfortunately. @priteshrnandgaonkar is our iOS wizard. He's on vacation right now but if this is still a problem next week, I'm sure he can chime in.

Reply:
@safaiyeh Thanks, alas those don’t explain _why_ they need to be build statically. I’ll wait to hear from @fkgozali 👍

Reply:
So Flipper has support for use_frameworks and the instructions above include instructions for use_frameworks, I think we just need to add the right handling and test

Reply:
@rickhanlonii I'll try that out today & update the template w/ it.